docs/_index.md

```diff
@@ -3,32 +3,32 @@ title: "Welcome to Knative"
 linkTitle: "Documentation"
 weight: 10
 type: "docs"
+showlandingtoc: "false"
 aliases:
    - /docs/concepts/overview.md
 ---
 
-The Knative project provides a set of [Kubernetes controllers](https://kubernetes.io/docs/concepts/architecture/controller/) that help to simplify developer and administrator experiences on [Kubernetes](https://kubernetes.io) clusters, by introducing an event-driven, serverless architecture.
+The Knative project provides a set of [Kubernetes](https://kubernetes.io) components that introduce event-driven, serverless capabilities for Kubernetes clusters.
 
-There are two core Knative components, that can be installed and used together or independently to provide different functions:
+Knative APIs build on existing Kubernetes APIs, so that Knative resources are compatible with other Kubernetes-native resources, and can be managed by cluster administrators using existing Kubernetes tools.
 
-* [Knative Serving](https://knative.dev/docs/serving/): Run stateless services more easily on Kubernetes,
-   by making autoscaling, networking and rollouts easier.
-
-* [Knative Eventing](https://knative.dev/docs/eventing/): Create subscriptions to event sources declaratively,
-   and route events to Kubernetes endpoints.
-
-These components are delivered as Kubernetes custom resource definitions (CRDs), which can be configured by a cluster administrator to provide default settings for developer-created applications and event workflow components.
-
-**Note**: Earlier versions of Knative included a Build component.  That component has been broken out into its own
-project, [Tekton Pipelines](https://tekton.dev/).
-
-Knative registers its own API types to the Kubernetes API, enabling developers and cluster administrators to use familiar languages and frameworks to solve common use cases, such as:
+Common languages and frameworks, which include Kubernetes-friendly tooling, work smoothly with Knative to reduce the time spent solving common deployment issues, such as:
```

Comment:
I think putting a `, which` here confuses things -- I think we mean "languages and frameworks which include K8s-friendly tooling" in contrast to languages and tools which don't have K8s-friendly tooling. (For example, Bash doesn't have a whole lot of tools to really help with K8s.)

This probably means removing both commas, since I think the "include K8s-friendly tooling" was meant as an aside, but in this case I think it's a qualifier instead.

Reply:
lol actually similar to the other comment on the other PR, but in this case it would then be "that", not "which". That is effectively the difference between the two; "that" is for a specific, restrictive clause, "which" is for non-restrictive clauses.
Good point though again, thanks for the attention to detail :heart_eyes: 

https://developers.google.com/style/pronouns?hl=en#relative-pronouns